app/lib/hesa/code_sets/age_ranges.rb

```diff
@@ -3,21 +3,21 @@
 module Hesa
   module CodeSets
     module AgeRanges
-      # https://www.hesa.ac.uk/collection/c21053/xml/c21053/c21053codelists.xsd
-      # https://www.hesa.ac.uk/collection/c21053/e/ittphsc
+      # https://www.hesa.ac.uk/collection/c22053/xml/c21053/c21053codelists.xsd
```

Comment:
This is a dead link for me:
<img width="538" alt="image" src="https://user-images.githubusercontent.com/420873/184145515-5cbac4d3-e33d-4eb5-a394-f71de1e385a2.png">
I also tried logging in to the datacollection subdomain and saw the same.

Reply:
thanks! i noticed that actually i needed to update all the mapping files for this, even where there have been no changes.

Addressed in https://github.com/DFE-Digital/register-trainee-teachers/pull/2632/commits/c9f0e3161787357ea5383dae6718485f6f9429a9